File: Latihan B/C++/Product.cpp
//kelas yang digunakan untuk mengimplementasikan kelas Product
class Product{

	private:

		//atribut private
		int price;
		string idProduct;

	public:

		//konstruktor
		Product(int price, string idProduct){
			this->price = price;
			this->idProduct = idProduct;
		}

		Product(){
			this->price = 0;
			this->idProduct = "";
		}

		//getter dan setter price
		void setPrice(int price){
			this->price = price;
		}

		int getPrice(){
			return this->price;
		}

		//getter dan setter idProduct
		void setIdProduct(string idProduct){
			this->idProduct = idProduct;
		}

		string getIdProduct(){
			return this->idProduct;
		}

		//destruktor
		~Product(){
			
		}

};

File: Latihan B/PHP/Product.php
<?php

//kelas yang digunakan untuk mengimplementasikan kelas Product
class Product{

	//atribut private
	private $price;
	private $idProduct;

	//konstruktor
	public function __construct($price = 0, $idProduct = ""){
		$this->price = $price;
		$this->idProduct = $idProduct;
	}

	//getter dan setter untuk atribut price
	public function setPrice($price){
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	//getter dan setter untuk atribut idProduct
	public function setIdProduct($idProduct){
		$this->idProduct = $idProduct;
	}

	public function getIdProduct(){
		return $this->idProduct;
	}

	//destruktor
	public function __destruct(){

	}

}

?>